kong/tracing/propagation.lua

```diff
@@ -343,6 +343,15 @@ local function find_header_type(headers)
   local b3_single_header = headers["b3"]
   if not b3_single_header then
     local tracestate_header = headers["tracestate"]
+
+    -- handling tracestate header if it is multi valued
+    if tracestate_header and type(tracestate_header) == "table" then
+      -- https://www.w3.org/TR/trace-context/#tracestate-header
+      -- Handling multi value header : https://httpwg.org/specs/rfc7230.html#field.order
+      tracestate_header = table.concat(tracestate_header, ',')
+      kong.log.warn("Trace state header is table : " .. tracestate_header)
```

Comment:
If it's expected behavior, then should not log a warn level msg.
```suggestion
      kong.log.debug("header `tracestate` is a table")
```

Reply:
done